frontend/src/pages/modelServing/screens/projects/ModelServingPlatform.tsx

```diff
@@ -153,19 +139,21 @@ const ModelServingPlatform: React.FC = () => {
           shouldShowPlatformSelection || platformError || emptyModelServer
             ? undefined
             : [
-                <ModelServingPlatformButtonAction
-                  isProjectModelMesh={isProjectModelMesh}
-                  testId={`${isProjectModelMesh ? 'add-server' : 'deploy'}-button`}
-                  emptyTemplates={emptyTemplates}
-                  onClick={() => {
-                    setPlatformSelected(
-                      isProjectModelMesh
-                        ? ServingRuntimePlatform.MULTI
-                        : ServingRuntimePlatform.SINGLE,
-                    );
-                  }}
-                  key="serving-runtime-actions"
-                />,
+                !isKServeNIMEnabled && (
```

Comment:
You could add this check to the previous OR list, no?

Reply:
Updated